vignettes/SEmIRD.Rmd

```diff
@@ -0,0 +1,316 @@
+
+\begin{equation}
+p_E(t) = \exp(-\kappa E t) \binom{E_0}{E} (1 - \exp(-\kappa t))^{E_0-E}.
+\end{equation}
+
+The mean number of exposed individuals at time $t$ is then given by:
+
+\begin{align}
+\bar{E}(t) &= \sum_{E=0}^{E_0} E p_E(t)\\
+&= \sum_{E=0}^{E_0} E \exp(-\kappa E t) \binom{E_0}{E} (1 - \exp(-\kappa t))^{E_0-E}\\
+&= E_0 \exp(-\kappa t) \sum_{E=1}^{E_0} \exp(-\kappa (E-1) t) \binom{E_0-1}{E-1} (1 - \exp(-\kappa t))^{(E_0-1)-(E-1)}\\
+&= E_0 \exp(-\kappa t),
+\end{align}
+
+where the third line is obtained by using $\binom{E_0}{E} = \frac{E_0}{E}\binom{E_0-1}{E-1}$; the last line is obtained since the sum extends over all possible values of $E$ for a distribution starting at $E_0-1$, which must equal 1.
+
+The most notable aspect of $\bar{E}(t)$ is that it matches the solution of the deterministic system. This shows that, for this system, there is an equivalence between the deterministic model and the stochastic model, where the former is a mean-field solution of the latter. This is nice because it allows us to ascribe "individual meaning" to parameters in the ODE: here, $1/\kappa$ is the mean duration spent in the infected state for an analogous stochastic model.
+
+## The latent period for COVID-19
+But what does the distribution governing the latent period actually look like? Using repeatedly sampled PCR test data from close contacts of confirmed cases in mainland China, a recent study estimated the latent period of COVID-19 as 5.5 days and a log-normal probability distribution was found to represent that waiting period [@xin2021estimating]. Below we compare the waiting period for the log-normal distribution fit in @xin2021estimating with an exponential distribution with the same latent period.
+
+```{r, fig.height = 5, fig.width = 7}
+# values derived from Xi et al., Table S2 
+mu <- 1.61
+sigma <- 0.44
+
+# equivalent decay for exponential model
+kappa <- 1 / 5.5
+
+# generate densities across rate of time
+t <- seq(0, 20, 0.01)
+lnorm_vals <- dlnorm(t, mu, sigma)
+exp_vals <- dexp(t, kappa)
+
+tibble(t, value=lnorm_vals, model="log-normal") %>% 
+  bind_rows(tibble(t, value=exp_vals, model="exponential")) %>% 
+  ggplot(aes(x=t, y=value, colour=model)) +
+  geom_line() +
+  scale_color_brewer("Model", palette = "Dark2") +
+  ylab("density") +
+  xlab("time (days)")
+```
+
+Although these share a common mean waiting period, there are considerable differences between the curves. An alternative way to view this is through the cumulative distribution functions:
+
+```{r, fig.height = 5, fig.width = 7}
+lnorm_vals <- plnorm(t, mu, sigma)
+exp_vals <- pexp(t, kappa)
+
+tibble(t, value=lnorm_vals, model="log-normal") %>% 
+  bind_rows(tibble(t, value=exp_vals, model="exponential")) %>% 
+  ggplot(aes(x=t, y=value, colour=model)) +
+  geom_line() +
+  scale_color_brewer("Model", palette = "Dark2") +
+  ylab("density") +
+  xlab("time (days)")
```

Comment:
Again these are covered in the SEIRD vignette. Is it because there is a demonstration between SEmIRD with SEIRD later on so that some introductions of SEIRD are included here?

Reply:
See above -- they're now only here.